### PR TITLE
Add "continue-on-error" CI machines to test automotive scanners

### PR DESCRIPTION
**Pull request by polybassa**
Hi, I would appreciate having the automotive scanners tests be part of the CI pipeline. However, I understand that some flaky tests cause problems for the overall pipeline. 

Would it be possible, to have some CI-machines for the automotive scanners only, which are allowed to fail?

### CODE REVIEW

**Comment by p-l-** [comment]
Hi @polybassa can you please rebase against current master? :pray:

**Comment by polybassa** [comment]
Done

**Comment by polybassa** [comment]
Ok.. just the codecov returned an error. All other CI-machines returned passed, even if a scanner-test failed

**Comment by polybassa** [comment]
@gpotter2 What are your thoughts about this PR?

**Comment by gpotter2** [comment]
This is a decent idea. (sorry for the delay)
I'd prefer to be able to keep the matrix clean though (for github ci). Do you think we could just add something similar to appveyor's UT_FLAGS but for that too ?

**Comment by polybassa** [comment]
Thanks for your response.

I'm not quite sure if I understand your request correctly. 

For GitHub you would prefer a test without matrix?

For appveyor, you ask if we can integrate the allow to fail in UT_FLAGS?


-- 
Viele Grüße

Nils Weiß

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -137,6 +137,48 @@ jobs:
     - name: Run tests
       run: tox -e cryptography
 
+  utscapy-scanner:
```
Comment (gpotter2) [inline .github/workflows/unittests.yml:140]: I believe it should be able to integrate this test with the other one, using something like
```
continue-on-error: ${{matrix...}
.......
UTscapy: ...
```

**Comment by polybassa** [comment]
@gpotter2 Do you like it more, now?

**Comment by gpotter2** [comment]
It's much better ! I see you bumped into the weird case where undefined values don't really work as expected...

**Comment by polybassa** [comment]
Yes, apparently the online example on GitHub was not working as intended. 
-- 
Viele Grüße

Nils Weiß

**Review by gpotter2 — APPROVED** [review]

**Comment by polybassa** [comment]
Thank you for your patience 
-- 
Viele Grüße

Nils Weiß

**Pull request merged**